src/test/clojure/xtdb/pgwire_test.clj

```diff
@@ -1592,6 +1592,7 @@
       (jdbc/execute! tx ["INSERT INTO foo RECORDS ?" (xt-jdbc/->pg-obj {:xt/id 4, :a "four"})])
 
       ;; HACK.
+      #_
```

Comment:
Commented out for now, as this didn't work. I suspect the flushing behavior might be the cause of this.